app/reducers/snackbar.js

```diff
@@ -609,6 +611,15 @@ const messages = defineMessages({
     defaultMessage:
       "Your vote has been cast!\nThanks for participating in Decred's governance"
   },
+  SETTREASURY_POLICY_SUCCESS: {
+    id: "setTreasuryPolicy.updateVoteChoiceSuccess",
+    defaultMessage:
+      "Your vote has been cast!\nThanks for participating in Decred's governance"
```

Comment:
Your treasury policy has been successfully updated! Thanks for participating in Decred's governance.